tests/containers/podman_netavark.pm

```diff
@@ -64,16 +64,20 @@ sub _cleanup {
     validate_script_output('podman network ls', sub { /podman\s+bridge/ });
 }
 
-sub switch_to_netavark {
-    my @pkgs = qw(netavark aardvark-dns);
+sub install_pkg {
```

Comment:
Please make a separate commit for this change.

Reply:
done